### DISCUSSION TITLE
How to get the font_name and font size of a page on scanned PDF documents

### DISCUSSION THREAD

**Discussion by Prabakaran-psl** [pymupdf/PyMuPDF#3958] [Looking for help]
**How to get the font_name and font size of a page on scanned PDF documents?**

I have used blocks > lines > spans > size for extracting text based pdf, but looking similar for scanned pdf document. Can someone please help me here?

**Comment by JorjMcKie** [comment]
There is no way in the world to achieve this. It has nothing to do with whatever tool you use: this information simply is not there.

**Reply by JorjMcKie** [reply]
It all depends on how clever the scanner / OCR-tool are which created the PDF. I know that e.g. ABBYY does a decent job with guessing the an approximate font.

Whatever:
Once the PDF is created, its content is all you have.
